nixos/doc/manual/README

```diff
@@ -1,12 +0,0 @@
-To build the manual, you need Nix installed on your system (no need
-for NixOS). To install Nix, follow the instructions at
-
-    https://nixos.org/nix/download.html
-
-When you have Nix on your system, in the root directory of the project
-(i.e., `nixpkgs`), run:
-
-    nix-build nixos/release.nix -A manual.x86_64-linux
-
-When this command successfully finishes, it will tell you where the
-manual got generated.
```

Comment:
I liked having the documentation right there, especially given it's not just a straightforward 'make'. Perhaps at least leave a link to the manual?

Reply:
Sorry, I don't understand. Are you missing the README?

Reply:
Yes, I think it's nice to have a reminder of the command close at hand when you're at the CLI already. Could also be a link to the docbook to reduce duplication, though.

Reply:
OK, then I replaced it's contents with a human link to the new location of this information instead of deleting it.

Reply:
Thanks, that works for me :+1: